app/controllers/conference_registrations_controller.rb

```diff
@@ -89,9 +87,9 @@ def destroy
       redirect_to root_path,
                   notice: "You are not registered for #{@conference.title} anymore!"
     else
-      redirect_to root_path,
-                  error: "Could not update your registration for #{@conference.title}: "\
-                  "#{@registration.errors.full_messages.join('. ')}."
+      redirect_to conference_conference_registrations_path(@conference.short_title),
```

Comment:
> Why do you redirect to conference_conference_registrations_path?

on GET request `conference_conference_registrations_path` goes to show action.
```
conference_conference_registrations_path	    POST	/conference/:conference_id/register(.:format)	conference_registrations#create
new_conference_conference_registrations_path	GET	/conference/:conference_id/register/new(.:format)	conference_registrations#new
edit_conference_conference_registrations_path	GET	/conference/:conference_id/register/edit(.:format)	conference_registrations#edit
                                                GET	/conference/:conference_id/register(.:format)	conference_registrations#show
                                                PATCH	/conference/:conference_id/register(.:format)	conference_registrations#update
                                                PUT	/conference/:conference_id/register(.:format)	conference_registrations#update
                                                DELETE	/conference/:conference_id/register(.:format)	conference_registrations#destroy
```
**show** method:
```Ruby
def show
    @workshops = @registration.workshops
    @total_price = Ticket.total_price(@conference, current_user)
    @tickets = current_user.ticket_purchases.where(conference_id: @conference.id)
  end
```
I think we need to change the name of `def show` to `def index`?